tests/test_email_account_app_password_integration.py

```
@@ -86,42 +118,16 @@ def test_account_email_processor_can_access_app_password(self):
-        fake_fetcher.get_recent_emails = Mock(return_value=[])
-        fake_fetcher.summary_service = Mock()
-        fake_fetcher.summary_service.clear_tracked_data = Mock()
-        fake_fetcher.summary_service.start_processing_run = Mock()
-        fake_fetcher.summary_service.complete_processing_run = Mock()
-        fake_fetcher.summary_service.run_metrics = {'fetched': 0}
-        fake_fetcher.stats = {'deleted': 0, 'kept': 0}
-        fake_fetcher.account_service = None
+        # Create fake fetcher
+        fake_fetcher = self._create_fake_fetcher()
 
         def create_fake_fetcher(email, password, token):
```

Ruff (0.14.6)

Unused function argument: `email`

(ARG001)